src/App.jsx

```diff
@@ -19,11 +19,14 @@ export function App() {
 	 * to create and join a new list.
 	 */
 	const [listToken, setListToken] = useStateWithStorage(
-		'my test list',
+		null,
 		'tcl-shopping-list-token',
```

Comment:
Do we still need to have the tcl-shopping-list-token to load on start?

Reply:
Yes, for the useStateWithStorage hook to work you need to specify a value for storageKey.